hack/test-end-to-end.sh

```diff
@@ -164,23 +164,25 @@ OS_PID=$!
 
 if [[ "${API_SCHEME}" == "https" ]]; then
 	export CURL_CA_BUNDLE="${CERT_DIR}/master/root.crt"
+  export CURL_CERT="${CERT_DIR}/admin/cert.crt"
```

Comment:
Tabs in this file are two spaces (chosen before me).  You want me to change them all?